logger/hooks/hooks.go

```diff
@@ -0,0 +1,42 @@
+// This file is part of Graylog.
+//
+// Graylog is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Graylog is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+
+package hooks
+
+import (
+	"path/filepath"
+
+	"github.com/rifflock/lfshook"
+	"github.com/Sirupsen/logrus"
+
+	"github.com/Graylog2/collector-sidecar/context"
+	"github.com/Graylog2/collector-sidecar/logger"
+)
+
+func AddLogHooks(context *context.Ctx, log *logrus.Logger) {
+	filesystemHook(context, log)
+}
+
+func filesystemHook(context *context.Ctx, log *logrus.Logger) {
+	logfile := filepath.Join(context.UserConfig.LogPath, "collector_sidecar.log")
+	writer := logger.GetRotatedLog(logfile, context.UserConfig.LogRotationTime, context.UserConfig.LogMaxAge)
```

Comment:
@jalogisch I had the same feeling in the beginning. At same point we started to create directories for the user to make the installation easier. So I guess we need it here as well. That's a all or nothing thing to be consistent with other configuration flags.